helpers/croplio.com.json

```diff
@@ -0,0 +1,12 @@
+{
+  "name": "Croplio",
+  "desc": "Transform your images instantly with free online crop tool",
+  "url": "https://croplio.com",
+  "tags": [
+    "Crop Image"
```

Comment:
Welcome to TH. Can you please not invent tags. :) The command `npm run helper:add` allows you to pick some tags.

Reply:
@wangbing-coder I'll fix things myself. Thanks for contributing to TH. :)